ExchangeSharp/API/Exchanges/ExchangeBinanceAPI.cs

```diff
@@ -533,7 +533,8 @@ protected override IDisposable OnGetOrderBookDeltaSocket(Action<ExchangeSequence
             decimal outputQuantity = await ClampOrderQuantity(symbol, order.Amount);
             decimal outputPrice = await ClampOrderPrice(symbol, order.Price);
 
-            payload["quantity"] = outputQuantity;
+            // Binance does not accept quantities with more than 20 decimal places.
+            payload["quantity"] = Math.Round(outputQuantity, 20);
```

Comment:
Why does ClampOrderQuantity fail in this case?

Reply:
ClampOrderQuantity doesn't consider the number of decimal places.
The symbols metadata does come with LOT_SIZE filter, but because it is treated as decimal, that information is lost. Perhaps there needs to be a new variable MaxDecimalPlaces that will be extracted from LOT_SIZE.